src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AuroraAzureAppSubPart.kt

```diff
@@ -33,14 +32,10 @@ val AuroraDeploymentSpec.azureAppGroups: List<String>?
     get() {
         //  Somewhat strange construction in order to avoid error with null as typeCast
         val rawGroup = this.getOrNull<Any>(AuroraAzureAppSubPart.ConfigPath.groups) ?: return null
+        @Suppress("UNCHECKED_CAST")
```

Comment:
Er det ikke mulig å heller gjøre slik:
```
val AuroraDeploymentSpec.azureAppGroups: List<String>?
    get() = this.getOrNull<List<String>>(AuroraAzureAppSubPart.ConfigPath.groups)
```

Reply:
Jeg ser at det gammel kode, men hvis vi ikke trenger den er det vel ikke vits å ha den.

Jeg testet selv å endre og ingen tester brakk. 
Jeg sjekket over testene og det finnes tester for alle permutasjoner av groups. Altså emptylist, ikke tilstede og satt.
Jeg tok også å debugget hvor groups ikke var satt. Da ble det returnert null med endringen min.